Gruntfile.js

```diff
@@ -355,6 +355,7 @@ module.exports = function(grunt) {
   grunt.registerTask('minify', ['bower','clean', 'build', 'minall']);
   grunt.registerTask('webserver', ['connect:devserver']);
   grunt.registerTask('package', ['bower','clean', 'buildall', 'minall', 'collect-errors', 'docs', 'copy', 'write', 'compress']);
-  grunt.registerTask('ci-checks', ['ddescribe-iit', 'merge-conflict', 'jshint', 'jscs']);
+  grunt.registerTask('ci-pre-checks', ['ddescribe-iit', 'jshint', 'merge-conflict']);
```

Comment:
Was re-arranging them inteded ? I think `jshint` will err if there are `merge` artifacts left over, so having `merge-conflict` run before `jshint`, might produce more meaningful errors (but I haven't really tested it :smiley:).